testprojects/src/python/interpreter_selection/BUILD

```diff
@@ -9,9 +9,7 @@ python_library(
   sources = ['echo_interpreter_version.py'],
   dependencies = [],
   # Play with this to test interpreter selection in the pex machinery.
-  # TODO(John Sirois): Allow `<4` when the issues with `3.7` are fixed. See:
-  #   https://github.com/pantsbuild/pants/issues/6363
-  compatibility = ['CPython>=2.7,<3.7']
+  compatibility = ['CPython>=2.7,<4']
```

Comment:
I noticed while debugging this that this line seems out of date now. But fixing it might be scope creep? Lmk if better to remove this from PR.